app/models/user.rb

```diff
@@ -0,0 +1,16 @@
+class User < ActiveRecord::Base
+  has_secure_password
+  has_many :bucketlists
+
+  validates_presence_of :firstname, :lastname, :email
+  validates_uniqueness_of :email
+  validates_presence_of :password, on: :create
+
+  before_save :generate_iss_number
+
+  private
+
+  def generate_iss_number
```

Comment:
Facilitator Feedback: Name should be changed to conform with local style guide and not JWT's.